db/migrations_tests/storagetagged_set_host_test.go

```diff
@@ -31,6 +31,10 @@ func TestStorageTaggedSetHost(t *testing.T) {
 	_, err := sqldb.Exec(`ALTER TABLE Deals ADD FastRetrieval BOOL`)
 	require.NoError(t, err)
 
+	// Add AnnounceToIPNI to allow tests to works
+	_, err = sqldb.Exec(`ALTER TABLE Deals ADD AnnounceToIPNI BOOL`)
+	require.NoError(t, err)
+
```

Comment:
Let's remove these two ALTER TABLE commands and instead add an INSERT like you did for deals_fast_retrieval_test.go